lib/components/atoms/FileInput/FileInput.js

```diff
@@ -0,0 +1,49 @@
+// @flow
+import React from 'react';
+
+import styled from 'styled-components';
+import styles from './FileInput.style';
+
+type Props = {
+  id: string,
+  className?: string,
+  type?: string,
+  name: string,
+  value?: string | number,
+  placeholder?: string,
+  ariaLabel?: string,
+  elementLocator?: string,
+  ref: ?HTMLInputElement,
+};
+
+const FileInput = React.forwardRef(
+  (
+    { id, className, type, name, value, placeholder, ariaLabel, elementLocator, ...others }: Props,
+    ref
+  ) => (
+    <input
```

Comment:
Why you not reusing exist Input component?

Reply:
Now, I have updated to utilize only one component.